src/styles/blocks/button.css

```diff
@@ -0,0 +1,18 @@
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  background-color: var(--design_color);
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+  color: var(--background_color_buy);
+}
+
+.card__button:hover {
+  color: var(--design_color);
+  background-color: var(--background_color_buy);
+  border: var(--border_color);
+}
```

Comment:
actually button is an element of the card block so it should be in the card.css file))